pnquickstart/ContentView.swift

```diff
@@ -62,97 +62,103 @@ struct ContentView: View {
 // MARK:- View Stores
 class PubNubStore: ObservableObject {
   @Published var messages: [Message] = []
-
+  
   var pubnub: PubNub
   let channel: String = "the_guide"
   let clientUUID: String = UUID().uuidString
-
+  
   init() {
-    var pnconfig = PubNubConfiguration(publishKey: "myPublishKey", subscribeKey: "mySubscribeKey")
+    var pnconfig = PubNubConfiguration(publishKey: "demo", subscribeKey: "demo")
```

Comment:
Suggest we put these back to placeholders, per our slack conversation...
```suggestion
    var pnconfig = PubNubConfiguration(publishKey: "myPublishKey", subscribeKey: "mySubscribeKey")
```